tools/travis/mypyer

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+script_dir=$(dirname "$0")
+lister_path="$script_dir/../lister.py"
+
+exclude="
+api/integrations/
+api/zulip/__init__.py
+bots/jabber_mirror_backend.py
+bots/zephyr_mirror_backend.py
+docs/conf.py
+puppet/zulip_internal/files/postgresql/pg_backup_and_purge.py
+tools/deprecated/
+tools/lister.py
+zproject/
+zerver/lib/actions.py
+zerver/lib/bugdown/__init__.py
+zerver/lib/bugdown/fenced_code.py
+zerver/lib/statistics.py
+zerver/migrations/
+zerver/models.py
+zerver/tests/
+zerver/views/__init__.py
+zerver/views/messages.py
+zilencer/models.py
+"
+
+python_files="$($lister_path --ext-only -f py --exclude $exclude)"
+# uncomment the line below to find out which files fail mypy check.
+#python_files="$($lister_path --ext-only -f py)"
+
+echo "$python_files" | xargs mypy --silent --py2 --check-untyped-defs
```

Comment:
It'd be good if this printed some human-readable output to both indicate when `mypy` starts running and when it finishes whether it was successful or not; currently it's hard to tell what happened from looking at the Travis logs of a success.

While addressing this, you should verify it still produces the right exit status